test/cypress/support/commands.js

```diff
@@ -90,6 +91,10 @@ Cypress.Commands.add('giLogin', (username, {
       }
       await sbp('gi.actions/identity/login', { username, password })
       await sbp('controller/router').push({ path: '/' }).catch(e => {})
+
+      return new Promise((resolve) => {
+        sbp('okTurtles.events/on', LOGIN, resolve)
+      })
```

Comment:
I don't think this should be here. And if it were needed for some reason, the promise should be setup before `'gi.actions/identity/login'` is called, not after, as that might miss the event being fired.

But I think this just shouldn't be here.

Reply:
Make sense. I thought I removed this one. Nice catch, @taoeffect

Reply:
Yeah it might have been from the merge of the 2 PRs